README.md

```diff
@@ -273,6 +279,22 @@ left untouched!",
 
         // inline arrays are just arrays with the correct type ...
         "inline_array_member1"  : [1, 2, 3],
+		
+		// an anyptr can be null, and then it is typeless
+		"anyptr_member1": null,
+		// but when it has a value then that will be stored in the "__subdata" section, and there will be a "__any_type_" entry right after the member.
+		"anyptr_member2": "ptr1234"
```

Comment:
This format will not really work if you have an array[any_pointer] or inline_array[any_pointer]... I would probably go with:
```json
"anyptr_member2" : {
    "type" : "ptr1234",
    "data" : {
        "x" : 1234,
        "y" : 5678
    }
}
```

that would also mesh better with any_array as that would be:
```json
"anyarray_member" : {
    "type" : "the_type",
    "data" : [ { "x" : 1, "y" : 2 }, { "x" : 4, "y" : 5 } ]
}
```

Reply:
Well arrays and inline arrays are functionally working today as you see in the tests, but I agree that they are not very pretty, and you can't put the elements wherever since the parser require them to be in order which might not be what you expect. So I might look into that when I get back to improving data_library. I've started on other tasks right now, so I will leaved as is this for a time now.

Reply:
If I find the time and energy for it I might have a go at it myself... do you agree with the proposed format above?

Reply:
We could maybe embed the type in the member-name... but that might be a bit "icky"... something like "my_any_member[the_type]" : { ... } ?

Reply:
... and it will not work for array-of-anyptr... back to the initial suggestion.

Reply:
I do like your initial suggestion.

Reply:
Fixed now